crown/src/kernel/boot.rs

```diff
@@ -47,13 +47,15 @@ pub fn setup(
 ) -> Result<NockApp, Box<dyn std::error::Error>> {
     let cli = cli.unwrap_or_else(|| Cli::parse());
 
-    tracing_subscriber::registry()
-        .with(
-            fmt::layer()
-                .with_ansi(cli.color == ColorChoice::Auto || cli.color == ColorChoice::Always),
-        )
-        .with(EnvFilter::new(&cli.log_level))
-        .init();
+    if !cfg!(feature = "skip-subscriber") {
+        tracing_subscriber::registry()
+            .with(
+                fmt::layer()
+                    .with_ansi(cli.color == ColorChoice::Auto || cli.color == ColorChoice::Always),
```

Comment:
Is there a way to easily query the color library's view of whether the terminal supports color or not, for the `ColorChoice::Auto` case?

Reply:
If it's not dead-simple to do this we'll file an issue and merge, not a blocker.

Reply:
there's a crate for it (`supports-color`), we could use that?

Reply:
https://github.com/zorp-corp/nada/issues/760